applications/luci-app-p910nd/po/pt_BR/p910nd.po

```diff
@@ -14,57 +14,194 @@ msgstr ""
 
-#: applications/luci-app-p910nd/luasrc/model/cbi/p910nd.lua:40
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:81
+msgid "Parallel port line printer device support"
+msgstr ""
+
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:121
 msgid "Port"
 msgstr "Porta"
 
-#: applications/luci-app-p910nd/luasrc/model/cbi/p910nd.lua:14
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:15
+msgid "Port 910n print daemon"
+msgstr ""
+
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:145
+msgid "Run as root"
+msgstr ""
+
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:166
+msgid "Serves as Location in Apple standards."
+msgstr ""
+
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:105
 msgid "Settings"
 msgstr "Configurações"
 
-# 20140621: edersg: tradução
-#: applications/luci-app-p910nd/luasrc/model/cbi/p910nd.lua:22
-msgid "Specifies the interface to listen on."
-msgstr "Especifica a interface para escuta."
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:227
+msgid "Some examples:"
+msgstr ""
 
-#: applications/luci-app-p910nd/luasrc/model/cbi/p910nd.lua:40
-msgid "TCP listener port."
-msgstr "Porta TCP de escuta."
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:158
+msgid "The %s type element."
+msgstr ""
+
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:174
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:203
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:218
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:226
+msgid "The %s value."
+msgstr ""
 
-#: applications/luci-app-p910nd/luasrc/model/cbi/p910nd.lua:18
-msgid "enable"
-msgstr "habilitado"
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:159
+msgid "User readable description of maker and model."
+msgstr ""
+
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:141
+msgid "Whether this print port is bi-directional."
+msgstr ""
+
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:150
+msgctxt "mDNS/Bonjour/ZeroConf"
+msgid "Whether to advertise this printer via %s."
+msgstr ""
+
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:132
+msgid "any"
+msgstr ""
+
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:175
+msgid "enclosed within parentheses"
+msgstr ""
+
+#: applications/luci-app-p910nd/htdocs/luci-static/resources/view/p910nd.js:65
+msgid "map ports 9100-9109 to local printers."
+msgstr ""
 
-#: applications/luci-app-p910nd/luasrc/model/cbi/p910nd.lua:9
 #: applications/luci-app-p910nd/root/usr/share/luci/menu.d/luci-app-p910nd.json:3
 msgid "p910nd - Printer server"
 msgstr "p910nd - Servidor de impressão"
 
+#~ msgid ""
```

Comment:
Is it possible to keep old messages?
If not then maybe we should just remove them

Reply:
If you can supply an automated way to do so which integrates into `i18n-update.pl` (which carries over existing translations for reuse), be my guest.